packages/emotion/src/index.js

```diff
@@ -82,7 +83,38 @@ export function css(objs: any, vars: Array<any>, content: () => Array<any>) {
   return computedClassName.trim()
 }
 
-function insertRawRule(css: string) {
+type inputVar = string | number
+
+export function customProperties(baseClassName: string, vars: Array<inputVar>) {
+  const hash = hashArray([baseClassName, ...vars])
+  const varCls = `css-vars-${hash}`
```

Comment:
I did this so that users could SSR and extract the custom property values, but I'm not sure if it will trip up in other places.